controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java

```diff
@@ -269,17 +269,6 @@ public void writeUpgradesPerMinute(double n) {
         curator.set(upgradesPerMinutePath(), ByteBuffer.allocate(Double.BYTES).putDouble(n).array());
     }
 
-    public Optional<Integer> readTargetMajorVersion() {
-        return read(targetMajorVersionPath(), ByteBuffer::wrap).map(ByteBuffer::getInt);
```

Comment:
Add a TODO for removing this path.

Reply:
Just forgot to remove it, fixed now :)

Reply:
https://github.com/vespa-engine/vespa/pull/23930